src/main/java/uk/gov/hmcts/reform/sscs/controller/CreateCaseController.java

```diff
@@ -0,0 +1,114 @@
+package uk.gov.hmcts.reform.sscs.controller;
+
+import static java.util.stream.Collectors.joining;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.security.SecureRandom;
+import java.util.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+import uk.gov.hmcts.reform.sscs.ccd.service.CcdService;
+import uk.gov.hmcts.reform.sscs.exception.SscsCorBackendException;
+import uk.gov.hmcts.reform.sscs.idam.IdamService;
+
+@RestController
+@ConditionalOnProperty("create_ccd_endpoint")
```

Comment:
Do you have a flux PR that sets CREATE_CCD_ENDPOINT value to false on prod?

Reply:
thanks, I am going to add this CREATE_CCD_ENDPOINT will raise PR for it